CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Contributors:
 
 ## dbt 0.19.0 (Release TBD)
 
+### Fixes
+
+- Fix exit code from dbt debug not returning a failure when one of the tests fail ([#3017](https://github.com/fishtown-analytics/dbt/issues/3017))
+
```

Comment:
@lynxcare Could you pop this changelog entry under `## dbt 0.20.0`, and add yourself to the list of contributors?